index.html

```diff
@@ -232,7 +232,7 @@ <h2>Nextcloud 28<a class="headerlink" href="#nextcloud-latest" title="Permalink
 							</div>
 
 							<div class="section" id="nextcloud-stable">
-								<h2>Nextcloud 27<a class="headerlink" href="#nextcloud-stable" title="Permalink to this headline">¶</a></h2>
+								<h2>Nextcloud 28<a class="headerlink" href="#nextcloud-stable" title="Permalink to this headline">¶</a></h2>
```

Comment:
after release?

Reply:
well after branch off I guess we would like to make clear that "latest" is not "28" anymore.